db/schema.rb

```diff
@@ -1051,14 +1051,15 @@
     t.datetime "updated_at", precision: 6, null: false
     t.datetime "discarded_at"
     t.string "get_an_identity_id"
+    t.string "archived_email"
+    t.datetime "archived_at"
     t.index ["discarded_at"], name: "index_users_on_discarded_at"
     t.index ["email"], name: "index_users_on_email", unique: true
     t.index ["get_an_identity_id"], name: "index_users_on_get_an_identity_id", unique: true
   end
 
   create_table "versions", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.string "item_type"
-    t.string "{:null=>false}"
```

Comment:
😬  not sure how that crept in

Reply:
I guess I am using different postgres version. Will fix it.

Reply:
I've put in a PR to fix it [here](https://github.com/DFE-Digital/early-careers-framework/pull/3639). Thanks for highlighting it!